front_end/src/app/(campaigns-registration)/projM123/page.tsx

```diff
@@ -0,0 +1,46 @@
+import ProfileApi from "@/services/profile";
+import Header from "./components/header";
+import { AlreadyRegisteredCard, ChoicesCards } from "./components/cards";
+import { redirect } from "next/navigation";
+import { Heading1, Heading2 } from "./components/typography";
+
+export const metadata = {
+  title: "projM123",
+  description: "Project M123",
+};
+
+export default async function Page() {
+  const user = await ProfileApi.getMyProfile();
+
+  if (user && user.registered_campaign_keys.indexOf("projM123") >= 0) {
+    return (
+      <>
+        <Header />
+        <main className="flex flex-grow justify-center">
+          <div className="mt-16 flex size-full flex-col items-center">
+            <Heading1>ProjectM123</Heading1>
+            <AlreadyRegisteredCard />
+          </div>
+        </main>
+      </>
+    );
+  } else if (user) {
+    redirect("register");
```

Comment:
Double-checking, is this a valid redirect? As far as I can see, we on't have `/register` route

Reply:
Tested and works, it's a relative path

Reply:
Apologies, I see the register route now. I must have missed it during my previous review for some reason.